test/cpp/naming/resolver_component_tests_runner_invoker.cc

```diff
@@ -135,3 +134,52 @@ void InvokeResolverComponentTestsRunner(std::string test_runner_bin_path,
 }  // namespace testing
 
 }  // namespace grpc
+
+DEFINE_bool(
```

Comment:
Please put these flag definitions up at the top of the file, right under the includes.

Reply:
done